src/struct_ls/pfmg.h

```diff
@@ -17,6 +17,18 @@
 //#define DEBUG_SETUP
 //#define DEBUG_SOLVE
 
+#define hypre_PFMGSetCIndex(cdir, cindex)       \
+   {                                            \
+      hypre_SetIndex(cindex, 0);                \
+      hypre_IndexD(cindex, cdir) = 0;           \
+   }
```

Comment:
I see this is just copy/paste (so no code changes), but I'm kind of curious about this function. It looks like it just sets the index to 0?